packages/server/graphql/types/Organization.ts

```diff
@@ -52,6 +54,16 @@ const Organization: GraphQLObjectType<any, GQLContext> = new GraphQLObjectType<a
         return isUserBillingLeader(viewerId, orgId, dataLoader)
       }
     },
+    totalMeetingCount: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: 'The number of meetings that have occurred in the org',
+      resolve: async ({id: orgId}, _args: unknown, {dataLoader}) => {
+        const allTeamsOnOrg = await dataLoader.get('teamsByOrgIds').load(orgId)
+        const teamIds = allTeamsOnOrg.map(({id}) => id)
+        const allMeetings = await dataLoader.get('allMeetingsByTeamId').loadMany(teamIds)
+        return allMeetings.filter(isValid).flatMap((meetings) => meetings).length
```

Comment:
Looks like we are fetching a lot of objects from the database to show a count. Some organizations might have a lot of meetings in total.
Does it make sense to run a single count query or cache the value in org table and increment?

Reply:
Hey @acressall, just wanted to bump this comment ^ as I think it got hidden in the barrage of GH notifications I've sent you

Reply:
@nickoferrall if we can drop it, this would be a solution. In other case if we want to avoid expensive migration, do you think it make sense to run a `.count()` query to rethink db instead of fetching all objects?

Reply:
To keep things moving, I've gone ahead and removed `Total Meetings` as it doesn't affect the price, which could be confusing for users, and it's expensive to calculate. 

@acressall if you feel strongly that we need to display `Total Meetings`, let me know and we can always add it back